src/Protocol/DFRN.php

```diff
@@ -3089,4 +3089,73 @@ private static function constructVerb(array $item)
 		}
 		return ACTIVITY_POST;
 	}
+
+	private static function tgroupCheck($uid, $item)
```

Comment:
What's a `tgroup`? What does this function?

Reply:
Don't know. It has something to do with forums.